tests/unit/h/cli/commands/init_test.py

```diff
@@ -1,92 +1,122 @@
-from unittest import mock
+import logging
+from unittest.mock import sentinel
 
 import pytest
+from sqlalchemy.exc import ProgrammingError
 
-from h.cli.commands import init as init_cli
+from h.cli.commands.init import init
 
 
-@pytest.mark.usefixtures("alembic_config", "alembic_stamp", "db", "search")
 class TestInitCommand:
-    @pytest.mark.skip(reason="We've been seeing flaky behavior on CI")
-    def test_initialises_database(
-        self, cli, cliconfig, db, db_engine, pyramid_settings
-    ):  # pragma: no cover
-        db.make_engine.return_value = db_engine
-        pyramid_settings["h.authority"] = "foobar.org"
+    def test_it_overrides_the_elasticsearch_client_timeout(self, cli, ctx, os):
+        result = cli.invoke(init, obj=ctx)
 
-        result = cli.invoke(init_cli.init, obj=cliconfig)
+        assert os.environ["ELASTICSEARCH_CLIENT_TIMEOUT"] == "30"
+        assert not result.exit_code
 
-        db.make_engine.assert_called_once_with(pyramid_settings)
+    def test_it_initializes_the_db(
+        self, alembic, caplog, cli, ctx, db, db_engine, pyramid_request
+    ):
+        caplog.set_level(logging.INFO)
+        db_engine.execute.side_effect = ProgrammingError("statement", "params", "orig")
```

Comment:
We're mocking the DB, so if we want to simulate what happens when the DB is not initialized (the `SELECT 1 FROM alembic_version` query fails) we just configure the mock DB engine to raise an exception.